### PR TITLE
Added Ecto and DB

### PR DESCRIPTION
**Pull request by jameskerr**

### CODE REVIEW

File: lib/hack_pop/parser.ex [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
-      |> Enum.map(&build_story/1)
+    Enum.map(list, &build_story/1)
   end
 
-  defp build_story({title, url, score}) do
-    %{title: title, url: url, score: score}
+  # PRIVATE
+
+  defp build_story({title, url, points}) do
+    %HackPop.Story{title: title, url: url, points: points}
```
Comment (jamiesunderland) [inline lib/hack_pop/parser.ex]: prefer a programming language i know

File: lib/hack_pop/pinger.ex [hunk 2]
```diff
@@ -2,7 +2,7 @@ require Logger
 
 defmodule HackPop.Pinger do
   @url "https://news.ycombinator.com"
-  @interval 5_000
+  @interval 60_000 * 5
```
Comment (jamiesunderland) [inline lib/hack_pop/pinger.ex]: 👍

File: test/hack_pop/parser_test.exs [hunk 3]
```diff
@@ -6,14 +6,16 @@ defmodule HackPopTest.ParserTest do
...
-  test "find_stories" do
+  test "find_stories parses title" do
     use_cassette "hackernews" do
       {:ok, response} = HTTPoison.get("https://news.ycombinator.com/")
       stories = HackPop.Parser.find_stories(response.body)
-      first = hd(stories)
-      assert first.title == "After 100 years World War I battlefields are poisoned and uninhabitable" 
-      assert first.url   == "http://www.wearethemighty.com/articles/after-100-years-world-war-i-battlefields-are-poisoned-and-uninhabitable"
-      assert first.score == 377
+      expected = %HackPop.Story{
```
Comment (jamiesunderland) [inline test/hack_pop/parser_test.exs]: 👍

File: lib/hack_pop/parser.ex (hunk 1 above)
Comment (jameskerr) [inline lib/hack_pop/parser.ex]: Sounds good, I'll make this change.

**Pull request merged**